kuma/wiki/tests/test_api.py

```diff
@@ -48,6 +48,9 @@ def test_doc_api(client, trans_doc):
         'title': 'Root Document',
         'url': '/en-US/docs/Root'
     }]
+    assert data['contributors'] == ['wiki_user']
```

Comment:
This assertion passes when I run the test locally, but is failing on Travis.

Reply:
Hmm. It only passes locally if I run kuma/wiki/tests/test_api.py in isolation. If I do `py.test kuma/wiki` to run all the wiki tests, it fails. So one of the other tests is somehow modifying the trans_doc or wiki_user fixtures

Reply:
I found a fix.